pkg/converter/source/virtualmachine_test.go

```diff
@@ -136,7 +136,7 @@ var _ = Describe("VirtualMachineConverter", func() {
 			accessor, _ = meta.Accessor(externalEntitySourcePatch.EmbedType())
 			_ = controllerruntime.SetControllerReference(accessor, ee, scheme)
 			for k, v := range externalEntitySourcePatch.GetTags() {
-				labels[k+config.ExternalEntityLabelKeyTagPostfix] = v
+				labels[config.ExternalEntityLabelKeyPrefix+k+config.ExternalEntityLabelKeyTagPostfix] = v
```

Comment:
declare a function for generating tag label.. Easy to change it if any update is required

Reply:
Done